HOWTO.md

```diff
@@ -0,0 +1,109 @@
+# Changes in recent Tigervnc versions
+Previous Tigervnc versions had a wrapper script called `vncserver`. This script
+could be run as a user manually to start *Xvnc* process. The usage was quite 
+simple as you just run:
+```
+$ vncserver :x [vncserver options] [Xvnc options]
+```
+and that was it. It worked just fine, however, there were issues when users
+wanted to use it in combination with *systemd*. Therefore, the implementation
+had to be changed to comply with *SELinux* and *systemd* rules.
+
+# How to start Tigervnc server
+## Add a user mapping
+With this you can map a user to a particular port. The mapping should be done in 
+`/etc/tigervnc/vncserver.users` configuration file. It should be pretty 
+straightforward. Once you open the file you will see there are some examples, but basically
+the mapping is in form:
+```
+:x=user
+```
+For example you can have
+```
+:1=test
+:2=vncuser
+```
+
+## Configure Xvnc options
+To configure Xvnc parameters, you need to go to the same directory where you did
+the user mapping and open `vncserver-config-defaults` configuration file. This 
+file is for the default Xvnc configuration and will be applied to every user 
+unless any of the following applies:
+* The user has its own configuration in `$HOME/.vnc/config`.
+* The same option with different value is configured in 
+  `vncserver-config-mandatory` configuration file, which replaces the default 
+  configuration and has even a higher priority than the per-user configuration.
+  This option is for system administrators when they want to force particular 
+  *Xvnc* options.
+
+Format of the configuration file is also quite simple as the configuration is
+in form of:
+```
+option=value
+option
+```
+for example:
+```
+session=gnome
+securitytypes=vncauth,tlsvnc
+desktop=sandbox
+geometry=2000x1200
+localhost
+alwaysshared
+```
+### Note:
+There is one important option you need to set and that option is the session you
+want to start. E.g when you want to start GNOME desktop, then you have to use:
+```
+session=gnome
+```
+This should match the name of a session desktop file from `/usr/share/xsessions`
+directory.
+
+## Set VNC password
+You need to set a password for each user in order to be able to start the 
+Tigervnc server. In order to create a password, you just run:
+```
+$ vncpasswd
+```
+You need to run it as the user who will run the server. 
+
+### Note:
+If you used Tigervnc before with your user and you already created a 
+password, then you have to make sure the `$HOME/.vnc` folder created by 
+`vncpasswd` have the correct *SELinux* context. You either can delete this 
+folder and recreate it again by creating the password one more time, or 
+alternatively you can run:
+```
+$ restorecon -RFv /home/<USER>/.vnc
+```
```

Comment:
Are we not backwards compatible with older folders?

Reply:
I'm not sure, I think they will not have correct context in case they were created before we installed our SELinux policy.

Reply:
That's not great and I would like a future patch to improve that. Many users will have existing folders with password files in them.

Reply:
I will try to look into that.